packages/react/src/common/useFocusByKeyboard.tsx

```diff
@@ -24,21 +24,18 @@ const FOCUSABLE_ELEMENTS = [
   'select:not([disabled])',
 ]
 
-type FocusWithArrowsOptions = {
-  ref: RefObject<HTMLDivElement>
+type FocusByKeyboardOptions = {
   rotating?: boolean
   directChildrenOnly?: boolean
   horizontally?: boolean
 }
 
-const useFocusWithArrows = ({
-  ref,
-  rotating = false,
-  directChildrenOnly = false,
-  horizontally = false,
-}: FocusWithArrowsOptions) => {
+export const useFocusByKeyboard = (ref: RefObject<HTMLDivElement>, options: FocusByKeyboardOptions) => {
```

Comment:
Wait… is it correct that we use `HTMLDivElement` here? Why div?

Reply:
No, I don't think so, should be `HTMLElement` I think

Reply:
Hm, does changing it to `HTMLElement` cause TypeScript errors? I think maybe the 'correct' way to do this is to [use a generic and pass the type to the function when calling it](https://stackoverflow.com/questions/57201223/typescript-how-to-declare-a-type-that-includes-all-types-extending-a-common-typ#answer-57201395), but that might be a bit much for now.

Reply:
I think I’ve applied that correctly; can you verify?

Reply:
Hm, I don't this this is any different from what we had? `ref: RefObject<HTMLElement>` seems to work for all types of html elements, and `<T extends HTMLElement>(ref: RefObject<T>)` seems to do the same thing.

The first option is good enough imo, and a bit simpler than the second one.

Reply:
I did get the TS error I quoted, but not anymore now. Maybe some other change. Reverted the generic.